<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-160</x>
      <y>60</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>180</y>
      <width>431</width>
      <height>321</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 38pt &quot;华文行楷&quot;;

color: rgb(255, 255, 0);
</string>
    </property>
    <property name="text">
     <string>海底钓鱼总动员</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_help">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>590</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/i/res/item1.png);
color: rgb(0, 85, 255);
font: 24pt &quot;华文行楷&quot;;
</string>
    </property>
    <property name="text">
     <string>帮助</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_exit">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>590</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(10, 255, 2);
border-image: url(:/i/res/item7.png);
font: 24pt &quot;华文行楷&quot;;
</string>
    </property>
    <property name="text">
     <string>退出游戏</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_start">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>590</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 0);
font: 24pt &quot;华文行楷&quot;;
border-image: url(:/i/res/anniu2.png);</string>
    </property>
    <property name="text">
     <string>开始游戏</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>921</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/i/res/help.PNG);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_back">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>680</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
border-image: url(:/i/res/item9.png);
font: 24pt &quot;华文行楷&quot;;
</string>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
